### PR TITLE
Add guest and staff actions/animations to the API

### PR DESCRIPTION
**Pull request by Manticore-007**
I've added actions/animations of the guests and staff members to the API. Lots of sandbox players love the create scenes with frozen peeps and a lot (including myself) would find it very convenient to trigger certain animations by hand. I want to include this functionality in the Peep Editor plugin.

See feature request #17965.

It is my first PR and I'm not a professional coder unfortunately. Please be gentle and I am open to learn.

### CODE REVIEW

File: src/openrct2/scripting/bindings/entity/ScStaff.cpp [hunk 1]
```diff
@@ -145,6 +161,34 @@ namespace OpenRCT2::Scripting
...
+
+    void ScStaff::action_set(const std::string value)
+    {
+        ThrowIfGameStateNotMutable();
+        auto peep = GetStaff();
+        if (peep != nullptr)
+        {
+            if (!StaffActionMap.TryGet(value))
+            {
+                return;
```
Comment (ZehMatt) [inline src/openrct2/scripting/bindings/entity/ScStaff.cpp:182]: I think it would be good if this throws an error rather than being a silent failure.

Comment (Manticore-007) [reply src/openrct2/scripting/bindings/entity/ScStaff.cpp]: Is this a valid solution?

https://github.com/OpenRCT2/OpenRCT2/pull/21020/commits/0f64670e6540cb66ad9d702a6baaf9e442806a02#diff-34aab9baf955b169e0245bcfc39a571b29cf929eed6216b5d444899667e25aacR488

Comment (ZehMatt) [reply src/openrct2/scripting/bindings/entity/ScStaff.cpp]: No, use duk_error, instead of printf. You can look for existing uses of duk_error.

**Review by ZehMatt — APPROVED** [review]
Looks fine to me,

**Pull request closed without merging**